README.md

```diff
@@ -172,82 +100,92 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - 25 LEDs
   - Light sensor
   - Motion sensors
-  - Acceleration detection
+  - Acceleration detection including gesture detection
   - Temperature sensor
 
 ### Useful Links
 - Tutorials and Example Code for BBC micro:bit:
   - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
   - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
 
-### How to use
+### Keybindings
+- Push Button `A for A, B for B, C for A & B`
+- Refresh the simulator: `SHIFT + R`
+- Run the simulator: `SHIFT + F`
 
-#### I. Take a look at the "Device Simulator Express: Getting Started" Command.
-1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+## How to use
+
+To use Device Simulator Express, install the extension from the marketplace and reload VS Code.
+
+### I. Take a look at the "Device Simulator Express: Getting Started" Command.
+1. Type in `"Device Simulator Express: Getting Started"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
 <img alt='Getting Started' src='https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/getting_started.png'>
-2. Choose the `micro:bit` option from the dropdown.
-3. Read, copy and learn some of the things you can do with the simulator!
-   
-#### II. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+1. Choose the the device you want to play with from the dropdown.

```

Comment:
The counter is off too. I think that instead of roman numerals we can go with cardinal numbers (**1.**, **2.**) and then add further steps either by **1.1** or **a.**.

Reply:
I would prefer it too, but the indenting becomes weird. Let me know if you're okay with the update or if we should try to come up with another solution.

Reply:
Yeah, doing a quicks research that doesn't seem like it's supported by GitHub markdown D: